Sources/App/Features/Presentations/Controllers/PresentationAPIController.swift

```diff
@@ -42,7 +41,7 @@ struct PresentationAPIController: RouteCollection {
         presentation.$speaker.id = try speaker.requireID()
         presentation.$event.id = try event.requireID()
 
-        if let secondSpeakerID = input.secondSpeakerID, (input.enableSecondSpeaker ?? false) {
```

Comment:
this `input.enableSecondSpeaker` was the culprit, It would literally never be true regardless of me using checked, values, etc on the select, so pulled it

Reply:
I believe this is because the name value is missing from the checkbox control

Reply:
I'm happy with it being solved this way instead as it actually cleans up the state management rather than leaving dangling selections that are no longer valid